packages/components/src/chart-uplot/index.tsx

```diff
@@ -49,16 +49,126 @@ interface UplotChartProps {
 	data: uPlot.AlignedData;
 	options?: Partial< uPlot.Options >;
 	legendContainer?: React.RefObject< HTMLDivElement >;
+	solidFill?: boolean;
 }
 
 export default function UplotChart( {
 	data,
 	options: propOptions,
 	legendContainer,
+	solidFill = false,
 }: UplotChartProps ) {
 	const translate = useTranslate();
 	const uplot = useRef< uPlot | null >( null );
 	const uplotContainer = useRef( null );
+	const defaultFillColor = 'rgba(48, 87, 220, 0.075)';
```

Comment:
Just thinking: What can we do here to prepare this component for multiple data series in the future? An array of default fill colors, perhaps?